src/transaction-runner.coffee

```diff
@@ -183,6 +182,7 @@ class TransactionRunner
       origin: transaction.origin
       startedAt: transaction.startedAt # number in ms (UNIX timestamp * 1000 precision)
       request: transaction.request
+    logger.debug('Transaction runner emits \'test error\' because of an error in hooks, runHooksForData()', transaction.name, error)
```

Comment:
Same as above, I don't understand what is `functionName()` after comma. Applies to all similar outputs in this commit.